docs/Security-Events-Preview-Configuration.md

```diff
@@ -0,0 +1,67 @@
+# Configuration for collection of security solution logs (Preview)
+
+OMS supports collection of the following 3rd party security log types:
+- **Common Event Format (CEF) logs**
+- **Cisco ASA logs**
+
+##Configuration summary
+1. Install and onboard the OMS Agent for Linux
+2. Configure Syslog forwarding to send the required logs to the agent on UDP port 25225
+3. Place the agent configuration [file][1] on the agent machine in ```/etc/opt/microsoft/omsagent/conf/omsagent.d/```
+4. Restart the syslog daemon and the OMS agent
+
+
+##Detailed configuration
+1. Download the OMS Agent for Linux, version 1.2.0-25 or above
+	- [OMS Agent for Linux GA v1.2.0-25] (https://github.com/Microsoft/OMS-Agent-for-Linux/releases/tag/OMSAgent_GA_v1.2.0-25)
+
+2. Install and onboard the agent to your workspace as described here:
+  - [Documentation for OMS Agent for Linux](https://github.com/Microsoft/OMS-Agent-for-Linux)  
+
+3. Send the required logs to the OMS Agent for Linux
+  1. Typically the agent is installed on a different machine from the one on which the logs are generated.
+	Forwarding the logs to the agent machine will usually require several steps:
+	- Configure the logging product/machine to forward the required events to the syslog daemon (e.g. rsyslog or syslog-ng) on the agent machine.
+	- Enable the syslog daemon on the agent machine to receive messages from a remote system.
+	
+  2. On the agent machine, the events need to be sent from the syslog daemon to local **UDP port 25225**. The agent is listening for incoming events on this port.  
```

Comment:
25225 is also used in the Docker Container solution. We should probably change the default one to 25226 to avoid collision between the two ports